### PR TITLE
MM-48016 - show tip shown after channel + board with classic style; small code improvements

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR sets the tour tip shown after the channel + board creation to the classic style (backdrop + white background).

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48016

#### Related Pull Requests
n/a

#### Screenshots
Captura de Pantalla 2022-10-28 a las 21 52 35

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/app_bar/app_bar.scss [hunk 1]
```diff
@@ -58,6 +58,7 @@ $channel-view-padding: 63px;
 
             .app-bar__icon-inner,
             span:not(.pulsating_dot) {
+                // if we want to show a tourtip/pulsating dot in any of the app bar icons, these styles must be ommitted when span.pulsating_dot
```
Comment (pvev) [inline components/app_bar/app_bar.scss:61]: not necessary as part of this change, but I forgot to put this valid comment in a previous related PR.

File: e2e/cypress/tests/integration/channel/new_channel_with_board_spec.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ describe('Channel routing', () => {
         });
     });
 
-    it('MM-T5141 Channel URL validation for spaces between characters', () => {
+    it('MM-T5141 New Channel is created with an associated Board', () => {
```
Comment (pvev) [inline e2e/cypress/tests/integration/channel/new_channel_with_board_spec.js:41]: not necessary for this change, but the description I used when initially adding this test was not the most accurate, actually was the one I copied from a test I used as a template

File: e2e/cypress/tests/support/ui/channel.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ Cypress.Commands.add('uiCreateChannel', ({
     if (createBoard) {
         cy.get('#add-board-to-channel').should('be.visible');
         cy.findByTestId('add-board-to-channel-check').then((el) => {
-            if (!el.hasClass('checked')) {
+            if (el && !el.hasClass('checked')) {
```
Comment (pvev) [inline e2e/cypress/tests/support/ui/channel.js:31]: not part of this change, but while reviewing the code I noticed that it was worth also adding this validation to the e2e test

File: packages/components/src/tour_tip/tour_tip.tsx [hunk 4]
```diff
@@ -41,6 +41,7 @@ type Props = {
     zIndex?: number;
     className?: string;
     showBackdrop?: boolean;
+    tippyBlueStyle?: boolean;
```
Comment (pvev) [inline packages/components/src/tour_tip/tour_tip.tsx:44]: @JulienTant for using the tour tip in the blue style you need to use this parameter as true :)

**Review by JulienTant — APPROVED** [review]
LGTM! Thanks for saving the blue style and making it easier to use!

**Review by salmanmanekia — APPROVED** [review]
LGTM! 🚢 Thanks @pvev!

**Comment by pvev** [comment]
@annelieseklein friendly reminder on this one :)

**Review by annelieseklein — APPROVED** [review]
looks good! in that highlight of the boards icon, can we just add 2px on the top and bottom?
